includes/api/ApiQueryUserContributions.php

```diff
@@ -486,4 +442,14 @@ public function getHelpUrls() {
 	public function getVersion() {
 		return __CLASS__ . ': $Id$';
 	}
+
+	private function normalizeValues( $values ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "normalizeValues" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1144)